packit_service/worker/monitoring.py

```diff
@@ -56,9 +56,9 @@ def __init__(self):
             registry=self.registry,
         )
 
-        self.no_status_after_15_s = Counter(
-            "no_status_after_15_s",
-            "Number of PRs/commits with no commit status for more than 15s",
+        self.no_status_after_40_s = Counter(
+            "no_status_after_40_s",
+            "Number of PRs/commits with no commit status for more than 40s",
```

Comment:
I am wondering if we want to still measure the count specifically, since if I am not mistaken, currently everywhere (all the graphs/alerts,..) the `initial_status_time` is used (even though probably I introduced both of these metrics:D)

Reply:
oh, you're right, I haven't noticed, thank you.

Reply:
well since I already made the changes, I'd just keep it :D unless you won't me to drop it